core/lib/libmbedtls/include/mbedtls_config.h

```diff
@@ -0,0 +1,29 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
```

Comment:
We already have a `libmbedtls/include/mbedtls_config_kernel.h`, please put the content of this file there instead.

Reply:
OK. I will update it.